li-utils/src/main/java/com/linkedin/common/urn/ChartUrn.java

```diff
@@ -31,9 +31,10 @@ public String getChartIdEntity() {
   }
 
   public static ChartUrn createFromString(String rawUrn) throws URISyntaxException {
-    Urn urn = new Urn(rawUrn);
+    final Urn urn = new Urn(rawUrn);
     validateUrn(urn, ENTITY_TYPE);
-    String[] urnParts = urn.getContent().split(",");
+    final String content = urn.getContent();
+    final String[] urnParts = content.substring(1, content.length() - 1).split(",");
```

Comment:
What changed here and why?

Reply:
This is a bug fix. I realized that parsing logic is not correct. Content contains opening and closing parenthesis. We need to get rid of them.